jobs/bbr-credhubdb/spec

```diff
@@ -39,3 +39,6 @@ properties:
     default: true
   credhub.data_storage.tls_ca:
     description: "CA trusted for making TLS connections to targeted database server"
+  credhub.data_storage.hostname_verification.enabled:
+    description: "Toggle certificate host name verification. WARNING: Only use if your database does not include a matching hostname in their server certificate!"
```

Comment:
The term "hostname verification" is usually spelled without a space between "host" and "name".

Reply:
fixed!